server/routes/auth.js

```diff
@@ -1,9 +1,15 @@
 import { Router } from "express";
 import socialAuthRouter from "./socialAuth";
-import { userLogin } from "../controllers/user";
+import { userSignUp, verifyUser, userLogin } from "../controllers/user";
```

Comment:
please can you change verifyUser to like verifyUserAccount, verifyUser looks generic to me.